aptos-move/framework/aptos-framework/sources/TestCoin.move

```diff
@@ -162,20 +162,24 @@ module AptosFramework::TestCoin {
         borrow_global<Balance>(owner).coin.value
     }
 
+	public(script) fun transfer(from: signer, to: address, amount: u64) acquires Balance, TransferEvents {
+		transfer_internal(&from, to, amount);
+	}
+
```

Comment:
spacing looks off here, make sure spaces not tabs

Reply:
Oh, let me run the formatter